docs/source/reference/api-server/api-server-admin-deploy.rst

```diff
@@ -915,4 +915,6 @@ If all looks good, you can now start using the API server. Refer to :ref:`sky-ap
 
     Advanced: Cross-Cluster State Persistence <examples/api-server-persistence>
     Advanced: Use OAuth/Okta Proxy <examples/api-server-auth-proxy>
+    Advanced: API server metrics monitoring <examples/api-server-metrics-setup>
```

Comment:
Besides the setup page, could we add a small section with the screenshot of the metrics in some page, to show case the functionality, e.g. https://docs.skypilot.co/en/latest/getting-started/quickstart.html?

Reply:
@rohansonecha